roles.md

```diff
@@ -2,8 +2,125 @@
+
+**Access:**
+
+*   Committers are granted _triage_, _maintain_ and _write_ access to the github repositories which they have contributed to.
+
+**Additional expectations of committers:**
+
+*   Understand and apply our quality guidelines.
+*   Understand and abide by our license requirements.
+*   Periodically contribute to our project over a one year period.
+
+**Eligibility:**
+
+Before being accepted for this role, a person should display the following characteristics:
+
+*   Provide sustained, valuable contributions to the project, typically for three months or more. An example demonstration of this could be:
+    *   Submit pull requests which have been accepted and merged by other committers. Typically one large, three medium, or five small pull requests over three months.
+    *   Or provide equivalent contributions to our community, such as outreach, contributing to our forums, or coordinating activities. 
+*   Express an interest in ongoing participation.
+*   Acknowledge the committer expectations.
+
+## Project Steering Committee
+
+The Project Steering Committee (PSC) is the official managing body for our project. The committee is drawn from the community and is based on merit and interest.
+
+*   We aim to attract a diverse mix of personal characteristics, reflective of the community we represent.
+*   We wish to avoid having one organisation employing a controlling proportion of the PSC.
+
+A PSC chair shall be selected from the PSC, with the only official extra duty being that they have a deciding vote in the case of a tie.
+
+**Access:**
+
+*   PSC members may be granted admin access to our github repositories if they have a need for them.
+
+**Additional expectations of PSC members:**
+
+*   Consider opinions from within our community, and advocate for the best interests of our community when making decisions.
+*   Vote according to your personal best judgment, as opposed to voting on behalf of your employer.
+*   Vote promptly (usually within days) to motions being raised.
+*   Regularly participate in project activities.
+*   Help with whatever unglamourous work that is required to keep the project moving forward.
+*   Monitor conversations within our forums and contribute constructively where appropriate.
+*   Optionally attend weekly meetings.
+
+**Eligibility:**
+
+Before being accepted for this role, a person should display the following characteristics:
+
+*   Sustained track record displaying the characteristics expected of PSC members, typically while acting as a committer within our project, typically for six months or more.
+*   Express an interest in ongoing participation.
+*   Acknowledge the PSC member expectations.
+
+## Mentors
+
+Initiatives which are funded by The Good Docs Project should have a mentor.  
+
+**Expectations of mentors:**
+
+A mentor’s responsibilities typically includes:
+
+*   Mentor a worker in developing a proposal.
+*   Facilitate making connections with other members within the community.
```

Comment:
Feel like this is a two-way street.  Pairing Mentors <> Mentees can be a challenge, we should put for effort internally to help this happen as it won't develop organically.  Think we should create some sort of `team` or `members` structure inside of the good-docs github org.  And in it list the members and their roles.  and surface those that are mentors and available for mentoring.

Reply:
Pair this into the onboarding